engine/session.js

```diff
@@ -517,6 +541,18 @@ class Session {
         debug(`[${this._sessionId}]: Not a leader and first|second media sequence in a VOD is requested. Invalidate cache to ensure having the correct VOD.`);
         await this._sessionState.clearCurrentVodCache();
         this.isAllowedToClearVodCache = false;
+        const diffMs = await this._playheadState.get("diffCompensation");
```

Comment:
Adding this to store, so that follower nodes can also calculate a propper tick interval. 
Followers read this from store at the same time they read a new currentVod.